package.json

```diff
@@ -46,13 +46,15 @@
     "babel": "^6.5.2",
     "browser-headers": "^0.4.1",
     "chai": "^3.5.0",
+    "grpc": "^1.22.2",
+    "grpc-tools": "^1.8.0",
     "lodash": "^4.17.5",
     "lodash.isequal": "^4.5.0",
     "minimist": "^1.2.0",
     "mocha": "^5.2.0",
     "mocha-spec-json-output-reporter": "^1.1.6",
     "source-map-support": "^0.4.18",
-    "ts-node": "^5.0.1",
+    "ts-node": "^8.3.0",
```

Comment:
As I was working on this, it seemed that ts-node 5.0.1 wasn't catching type errors in test/integration/service/grpcnode.ts - I tried updating to the latest 8.3.0 and then it did catch them and would cause `npm run test` to fail, as expected.